ci-automation/tapfile_helper_lib.sh

```diff
@@ -77,21 +77,25 @@ function __db_init() {
 # 3: <run>     - re-run iteration
 
 function tap_ingest_tapfile() {
+
     local tapfile="${1}"
     local vendor="${2}"
     local run="${3}"
 
     local result=""
     local test_name=""
-    local error_message=""
+    local error_message_file="$(mktemp)"
     local in_error_message=false
 
     if ! [ -f "${TAPFILE_HELPER_DBNAME}" ] ; then
        __db_init
     fi
 
     # Wrap all SQL commands in a transaction to speed up INSERTs
+    # We will commit intermediately if there's an error message to insert so the
```

Comment:
Immediately? Intermittently? I think you meant the former. :)

Reply:
Actually, just googled "intermediate" and this does not seem to be a wrong word actually. TIL.